### ISSUE TITLE
NTR: Abnormality of the Puerperium

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#6217]
**Preferred term label:**
Abnormality of the Puerperium  

**Synonyms**
Puerperal period, Peripartum period, Immediate postpartum period, acute and subacute postpartum period

**Definition (free text, please give PubMed ID)**
Period after birth lasting up to 6 weeks, which is marks the return of a mother's body to a non-pregnant state. This phase is marked by rapid and major body changes impacting the genitourinary system, hemodynamics, metabolism, hormones, and emotionality. While concerns can be mild and self-identified, they can also be fast-evolving and life-threatening. This stage can be divided into acute (first 6-12 hours) and subacute (2-6 weeks). 

*Citation:* [PMC:3279173](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3279173/)

**Parent term (use hpo.jax.org/app)**
Abnormality of prenatal development or birth (`HP_0001197`). Choosing this term because it is currently the parent for Prenatal Maternal abnormality (`HP_0002686`).

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
- Postpartum Depression (`MONDO_0005929`)  
- Postpartum Psychosis (`MONDO_0018623`)  
- Uterine Inversion (`MONDO_0004936`)  
- Eclampsia (`MONDO_0001754`)   
- Peripartum Cardiomyopathy (`MONDO_0018920`)   

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-8169-9049

**Comment by pnrobinson** [comment]
@callahantiff  I am not sure whether it might be better to use a Clinical modifier for this. Many of the abnormalities that occur with increased frequency in this period can also occur at other times of life. What about a Puerperal onset term? https://hpo.jax.org/app/browse/term/HP:0031797

**Comment by callahantiff** [comment]
> @callahantiff I am not sure whether it might be better to use a Clinical modifier for this. Many of the abnormalities that occur with increased frequency in this period can also occur at other times of life. What about a Puerperal onset term? https://hpo.jax.org/app/browse/term/HP:0031797

Hi @pnrobinson - I think that this sounds like exactly the right solution! I wish I would have thought of that. 

In general, I am wondering how you decide on where to draw the line between making something a new class versus adding it as a modifier? In retrospect, I can totally see now how this type of concept fits better as a modifier onset term. Do you have any advice on how I can think about this moving forward? I think I know what you are going to say, but I am asking here anyway 😄

**Comment by pnrobinson** [comment]
@callahantiff  -- I guess it is a little subjective, but (1) we are tyring to avoid combinatorial explosions of terms and (2) we make an exception if some combination is distinctive. For instance, if the phenotype of puerperal depression could easily be distinguished from that of general depression. Many of the prenatal phenotypes are just not observed at all postnatally and so that seemed to justify having a separate section.
I will add puerpural onset as a modifer term

**Comment by callahantiff** [comment]
That sounds great!

**Comment by pnrobinson** [comment]
I added a term, 
Puerperal onset
def: Onset of a phenotypic abnormality or a disease during childbirth or in the period immediately following.
